src/Parentheses.cpp

```diff
@@ -1,6 +1,31 @@
 #include "Parentheses.hpp"
+#include <stack>
 
 bool Parentheses::isBalanced(std::string s) {
-   
-   return (s!=")(");
-}
\ No newline at end of file
+
+int i = 0;
+std::stack<char> ParenthesesStack{};
```

Comment:
taka sugestia tylko - to może być polem klasy. I nazwy zmiennych mała literą zaczynamy